src/components/AntdSelect/AntdSelect.js

```diff
@@ -332,20 +355,28 @@ const AntdSelect = props => {
                 </>
               ) : (
                 <Icon
+                  className="selectable-icon"
                   color="gray"
                   name="chevron_down"
                   size="small"
-                  onClick={e => {
+                  onClick={() => {
                     setShowDropdown(true);
-                    e.stopPropagation();
                   }}
                 />
               )
             }
-            open={showDropdown}
             ref={r => {
               ref.current = r;
             }}
+            onDropdownVisibleChange={e => {
```

Comment:
There's 2 times where you define onDropdownVisibleChange prop (line 278 and here, 371)

Reply:
I also applied it to the SingleSelector